### PR TITLE
Add essential files into the dist/ folder

### LINKED ISSUE #198: NPM page missing README.md file
**Description**
The `npm` page for Vocabulary states that a `README.md` file could not be found for the project, which is odd cause we do have one.

**Reproduction**
Visit [Vocabulary on npm](https://www.npmjs.com/package/@creativecommons/vocabulary).

**Expected behavior**
The `README.md` that appears on GitHub should appear.

**Screenshots**
Screenshot 2020-03-19 at 12 45 50 AM

**Additional context**
The fault is with the `deploy.sh` script.

**Want to take up?**
Yes

### PR DESCRIPTION
**Pull request by dhruvkb**
**Fixes**

Partially addresses #198 by @dhruvkb 

**Description**
Adds the missing `README.md` and `LICENSE` files into the dist directory before packing and publishing.

**Type of PR**

This PR is a hotfix.

**Technicalities**
Add a couple more `cp` commands to the `deploy.sh` script.

**Checklist:**
 
- [x] My pull request has a descriptive title (not a vague title like "Update `index.md`").
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

**Legal**
By submitting this PR, I agree to abide by the terms of the Developer 
Certificate of Origin.

<details>
<summary>Developer Certificate of Origin</summary>

Developer Certificate of Origin<br>
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.<br>
1 Letterman Drive<br>
Suite D4700<br>
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
</details>

### CODE REVIEW

**Comment by dhruvkb** [comment]
@brenoferreira I changed "Fixes" to "Partially addresses" because the issue with `npm` will not go away until another release.

File: deploy.sh [hunk 1]
```diff
@@ -12,6 +12,8 @@ trap '_es=${?};
 
 npm run build
 cp package.json dist/
+cp README.md dist/
+cp LICENSE dist/
```
Comment (brenoferreira) [inline deploy.sh:16]: NPM is able to figure out the license https://www.npmjs.com/package/@creativecommons/vocabulary

Do we need to copy the LICENSE file too?

Comment (hugosolar) [reply deploy.sh:16]: All our repos contain a license file, I think we should use the same criteria. Bulma includes a copy of the license too.

**Review by hugosolar — APPROVED** [review]

File: deploy.sh (hunk 1 above)
Comment (dhruvkb) [reply deploy.sh:16]: The files I picked are the exact same that `npm` picks in addition to the `files` key in the `package.json` file. Plus it's always good legal advise to include a copy of the license.

**Pull request merged**